### ISSUE TITLE
Menu disappears under cards

### ISSUE THREAD

**Issue by HrCalmar** [Kibibit/hass-kibibit-theme#22]
Hallo,

When I want to delete a card, the dialog box goes under the other cards.

![d2e29554-673f-4c4a-8605-205d69163c97](https://user-images.githubusercontent.com/17986805/104106429-213b6c00-52b6-11eb-8e54-0d245acd1b13.png)

/Calmar

**Comment by thatkookooguy** [comment]
@HrCalmar  Yes... That's a problem with [@kibibit/kb-frosted-cards](https://github.com/kibibit/kb-frosted-cards).

I'm trying to fix this but can't find an easy solution for now.

You can keep the theme itself without the `kb-frosted-cards` extension and it will work fine. the `kb-frosted-cards` extension is mainly for using other backgrounds than the default one. Since in the screenshot it seems like you're using the default background, the experience won't be much different without the `kb-frosted-cards` extension.
